src/OmniSharp.Abstractions/Models/v1/ReAnalyze/ReanalyzeRequest.cs

```diff
@@ -0,0 +1,13 @@
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Abstractions.Models.V1.ReAnalyze
+{
+    [OmniSharpEndpoint(OmniSharpEndpoints.ReAnalyze, typeof(ReAnalyzeRequest), typeof(ReanalyzeResponse))]
+    public class ReAnalyzeRequest: IRequest
```

Comment:
shouldn't this just be an empty class inheriting from `SimpleFileRequest`? then the filename will have consistent naming to other endpoints, plus it will get the same directory separator semantics